content/espanol/csharp-guess-the-word/activity-1-es.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Activitdad 1"
+draft: false
+weight: 4
+---
```

Comment:
We need to tell Hugo that this page is a translation of the English page. Two ways to do this:

1. (Preferred) Add a translation key in this file and in the corresponding English file. This is so that one can move between the English and Spanish page using the drop down.

Suggestion:
```suggestion
translationKey: "csharp-guess-the-word-activity1"
---
```

2. Rename this file to activity-1.md (using the same name as the English file). This does mean though that the URL for this page is https://workshops.nuevofoundation.org/csharp-guess-the-word/activity-1/ instead of maybe https://workshops.nuevofoundation.org/csharp-adivina-la-palabra/actividad-1/

Reply:
ok, I can work on this sometime this week. I will go with the preferred approach.